sktime/classification/feature_based/_fresh_prince.py

```diff
@@ -0,0 +1,206 @@
+# -*- coding: utf-8 -*-
+"""FreshPRINCE Classifier.
+
+Pipeline classifier using the full set of TSFresh features and a RotationForest
+classifier.
+"""
+
+__author__ = ["MatthewMiddlehurst"]
+__all__ = ["FreshPRINCE"]
+
+
+from sktime.classification.base import BaseClassifier
+from sktime.contrib.vector_classifiers._rotation_forest import RotationForest
+from sktime.transformations.panel.tsfresh import TSFreshFeatureExtractor
+from sktime.utils.validation.panel import check_X_y
+
+
+class FreshPRINCE(BaseClassifier):
+    """Fresh Pipeline with RotatIoN forest Classifier.
+
+    This classifier simply transforms the input data using the TSFresh [1]
```

Comment:
I believe you need an _ to reference so it should be:

`the input data using the TSFresh [1]_`